modules/services/gnome-keyring.nix

```diff
@@ -28,6 +28,14 @@ in {
     assertions = [
       (lib.hm.assertions.assertPlatform "services.gnome-keyring" pkgs
         lib.platforms.linux)
+      {
+        assertion = !config.services.pass-secret-store.enable;
```

Comment:
pass-secret-store does not exist, should be pass-secret-service?

Reply:
Looks like it. This merged PR causes the following error on master currently:
```
building the system configuration...
error: attribute 'pass-secret-store' missing

       at /nix/store/bn8h6s75bca34kilcmxgx9kpwz9d4xji-source/modules/services/gnome-keyring.nix:32:22:

           31|       {
           32|         assertion = !config.services.pass-secret-store.enable;
             |                      ^
           33|         message = ''
(use '--show-trace' to show detailed location information)
```

Reply:
*grumble* how did the tests not catch this

Reply:
Oh we don't have a test for gnome-keyring. That would be why.